RPM.spec.in

```diff
@@ -1,14 +1,14 @@
 # This is an RPM spec file that specifies how to package
 # ESS for Fedora Core Linux and, possibly, similar systems.
 
-%define name emacs-ess
```

Comment:
Why the name change?

Reply:
[we good now](https://github.com/emacs-ess/ESS/pull/732#issuecomment-439958230)

Reply:
It was looking for files in `emacs-ess` instead of `ess` which is where they were.  It's not too important as we are merely using RPM.spec as a coarse-grained test.